schema/src/main/scala/org/constellation/schema/NodeState.scala

```diff
@@ -36,11 +36,11 @@ object NodeState extends Enum[NodeState] with CirceEnum[NodeState] {
 
   val validForDownload: Set[NodeState] = Set(PendingDownload, Ready)
 
-  val validForRedownload: Set[NodeState] = Set(ReadyForDownload, Ready)
+  val validForInitialRedownload: Set[NodeState] = Set(ReadyForDownload)
```

Comment:
Isn't the initial redownload just download? Maybe we should rename the state listed in line 37 to something different.

Reply:
It kind of is. But we already have a val used in other places called validForDownload with different states inside. That's why I came up with this name. Do you have any suggestion how to call it better?

Reply:
Ahh maybe we could rename the one in 37, I will think about it!